modules/nf-core/deepvariant/rundeepvariant/main.nf

```diff
@@ -38,12 +45,13 @@ process DEEPVARIANT {
         --output_gvcf=${prefix}.g.vcf.gz \\
         ${args} \\
         ${regions} \\
-        --intermediate_results_dir=. \\
+        ${par_regions} \\
+        --intermediate_results_dir=tmp \\
```

Comment:
are we sure about this one? If nextflow starts writing to /tmp here againit will break a lot of cluster. The `.` should ensure it writes to the current scratch directory

Reply:
I'll update the module upstream

Reply:
https://github.com/nf-core/modules/pull/6730